flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/Fabric8ClientTest.java

```diff
@@ -187,6 +188,9 @@ public void testCreateFlinkMasterDeployment() {
 		assertEquals(new File(mountPath, FLINK_CONF_FILENAME).getPath(),
 			jmContainer.getVolumeMounts().get(0).getMountPath());
 		assertEquals(FLINK_CONF_FILENAME, jmContainer.getVolumeMounts().get(0).getSubPath());
+
+		assertEquals(1, jmContainer.getEnv().size());
+		assertEquals(new EnvVar(FLINK_MASTER_ENV_KEY, FLINK_MASTER_ENV_VALUE, null), jmContainer.getEnv().get(0));
```

Comment:
I think this can be expressed more succinctly by using `assertThat(jmContainer.getEnv(), contains(new EnvVar(FLINK_MASTER_ENV_KEY, FLINK_MASTER_ENV_VALUE, null)))`;

Reply:
Make sense to me. Using `assertThat` is better.